SygicUIKit/Classes/Views/Search/SYUISearchBarView.swift

```diff
@@ -23,13 +23,19 @@
 import UIKit
 
 
+/// Search bar view output delegate. Same delegate is used in `SYUISearchBarController`.
+/// So `SYUISearchBarView` can be subclassed and search UI delegate will be same.
```

Comment:
zasa vyplyva otazka ze preco ho nedat ako open ked uz piseme ze moze byt subclassnuty

Reply:
ale vlastne ten protokol je na to aby si ho prave nemusel subclassovat. naopak si mozes urobit vlastny searchBar a SYUISearchBar uplne odignorovat takze by som toto upravil ze to je protokol co definuje interface pre search bar view